Azure/WeatherStation.PowerBI/SensorWeatherDataProvider.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Devices.I2c;
+using Windows.Devices.Enumeration;
+
+namespace WeatherStation
+{
+    class SensorWeatherDataProvider : IWeatherDataProvider
+    {
+        public static async Task<SensorWeatherDataProvider> Create()
+        {
+            String aqs = I2cDevice.GetDeviceSelector("I2C1");
```

Comment:
nit: i think the newer APIs are more intuitive:

            var i2cController = await I2cController.GetDefaultAsync();
            var sensor = i2cController.GetDevice(new I2cConnectionSettings(0x40));